src/System Application/App/Privacy Notice/src/MicrosoftLearn/MicrosoftLearnPrivacyNotice.Codeunit.al

```diff
@@ -0,0 +1,35 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Privacy;
+
+codeunit 1569 "Microsoft Learn Privacy Notice"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        ShowingPrivacyNoticeTelemetryTxt: Label 'Showing privacy notice for Microsoft Learn', Locked = true;
+        TelemetryCategoryTxt: Label 'Privacy Notice Microsoft Learn', Locked = true;
```

Comment:
Minor, if the telemetry category is defined in AL I usually prefer to prefix it with AL, e.g. `AL Privacy Notice Microsoft Learn`.
I think around 60% of the AL code is doing this (which is a shame imho) but I still point it out whenever I spot it in PRs. But you are welcome to not fix it, I don't have a strong opinion :)